src/components/bs5/head/head.stories.js

```diff
@@ -0,0 +1,76 @@
+import defaultdata from "./head.data.json";
+import { Head } from "./Head.js";
+
+export default {
+  tags: ["autodocs"],
+  title: "!Core/HeadMeta",
+  render: (args) => {
+    return new Head(args).html;
+  },
+
+  argTypes: {
+    cdn: {
+      name: "CDN",
+      description: `CDN prefix or provided text`,
+      control: { type: 'radio' ,
+        labels: { 'DEV': 'DEV', 'TEST' : 'TEST','BETA':'BETA','STAGING':'STAGING','PROD':'PROD','/__data/assets/git_bridge/0026/471752/': 'SQUIZ Custom'},
+      },
+      options: [
+        "DEV",
+        "TEST",
+        "BETA",
+        "STAGING",
+        "PROD",
+        "/__data/assets/git_bridge/0026/471752/",
```

Comment:
The trailing slash on this Squiz custom option (or the leading slash in the handlebars template) is causing a double up in the output
![image](https://github.com/qld-gov-au/qgds-bootstrap5/assets/17775683/19f7be39-2f16-4067-a33a-8d435f897b98)

Reply:
fixed the story, it shows how a custom text can be used.